ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClusterDetailsPage.tsx

```diff
@@ -154,8 +154,7 @@ function ClusterDetailsPage() {
                         {isLoadingClusterProfileData ? (
                             <Skeleton screenreaderText="Loading cluster name" width="150px" />
                         ) : (
-                            // TODO: placeholder until we get cluster name
-                            clusterId
+                            clusterProfileData?.clusterName
```

Comment:
For brainstorming, not a blocker to merge, whether there is a pattern to help TypeScript know that `clusterProfileData.clusterName` is known in alternate of ternary expression.